editor/editor_node.cpp

```diff
@@ -6111,6 +6097,11 @@ EditorNode::EditorNode() {
 	_dim_timer->set_wait_time(0.01666f);
 	_dim_timer->connect("timeout", this, "_dim_timeout");
 	add_child(_dim_timer);
+
+	ED_SHORTCUT("editor/editor_2d", TTR("Open 2D Editor"), KEY_F1);
```

Comment:
Please also take the opportunity to put the shortcuts back to F2 for 2D and F3 for 3D, the current ones always bothered me :P
F1 should likely open the quick search dialog of the help.